packages/terra-hyperlink/src/Hyperlink.jsx

```diff
@@ -43,17 +43,25 @@ const propTypes = {
   /**
    * The content to display inside link.
    */
-  children: PropTypes.node,
+  text: PropTypes.string.isRequired,
   /**
    * Sets the href of the link.
    */
   href: PropTypes.string,
   /**
+   * @private
    * Whether or not the link should be disabled.
+   *
+   * ![IMPORTANT](https://badgen.net/badge/UX/Accessibility/blue) Accessibility best practices are to not use hyperlinks that are disabled.
```

Comment:
I thought private props would not appear in the documentation. If I'm correct then what is the purpose of including the badge here?

Reply:
We have hidden couple of examples from the docs page which will be presented to the users later in a better way where certain scenarios will be explained regarding where to use these props. So the badge might come in handy there.